.gitignore

```diff
@@ -11,6 +11,10 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
+# Files generated by JetBrains IDEs, e.g. IntelliJ IDEA
+.idea/
+*.iml
```

Comment:
@brianpursley I just noticed this.

Ideally we don't wanna have editor-specific stuff here.

You should add these to your ~/.gitignore_global file and set git global config `core.excludesfile=~/.gitignore_global`

Reply:
Sorry about that.  I will make a PR to clean it up :)